src/activities/Elsa.Activities.AzureServiceBus/Activities/SendAzureServiceBusMessage/AzureServiceBusSendActivity.cs

```diff
@@ -0,0 +1,100 @@
+using Microsoft.Azure.ServiceBus;
+using Microsoft.Azure.ServiceBus.Core;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Elsa.Activities.AzureServiceBus
+{
+    public abstract class AzureServiceBusSendActivity : Activity
+    {
+        protected ISenderClient Sender;
+
+        protected IContentSerializer _serializer;
+        
+        public AzureServiceBusSendActivity(IContentSerializer serializer)
+        {
+            _serializer = serializer;
+        }
+
+        [ActivityInput(SupportedSyntaxes = new[] { SyntaxNames.JavaScript, SyntaxNames.Liquid, SyntaxNames.Json })]
+        public object Message { get; set; } = default!;
+
+        [ActivityInput(SupportedSyntaxes = new[] { SyntaxNames.JavaScript, SyntaxNames.Liquid })] 
+        public string? CorrelationId { get; set; }
+        [ActivityInput(SupportedSyntaxes = new[] { SyntaxNames.JavaScript, SyntaxNames.Liquid })] 
+        public string ContentType { get; set; } = default!;
+        [ActivityInput(SupportedSyntaxes = new[] { SyntaxNames.JavaScript, SyntaxNames.Liquid })] 
+        public string? Label { get; set; }
+        [ActivityInput(SupportedSyntaxes = new[] { SyntaxNames.JavaScript, SyntaxNames.Liquid })] 
+        public string? To { get; set; }
+        [ActivityInput(SupportedSyntaxes = new[] { SyntaxNames.JavaScript, SyntaxNames.Liquid })] 
+        public string MessageId { get; set; } = default!;
+        [ActivityInput(SupportedSyntaxes = new[] { SyntaxNames.JavaScript, SyntaxNames.Liquid })] 
+        public string? PartitionKey { get; set; }
+        [ActivityInput(SupportedSyntaxes = new[] { SyntaxNames.JavaScript, SyntaxNames.Liquid })] 
+        public string? ViaPartitionKey { get; set; }
+        [ActivityInput(SupportedSyntaxes = new[] { SyntaxNames.JavaScript, SyntaxNames.Liquid })] 
+        public string? ReplyTo { get; set; }
+        [ActivityInput(SupportedSyntaxes = new[] { SyntaxNames.JavaScript, SyntaxNames.Liquid })] 
+        public string? SessionId { get; set; }
+        [ActivityInput(SupportedSyntaxes = new[] { SyntaxNames.JavaScript, SyntaxNames.Liquid })] 
+        public DateTime ExpiresAtUtc { get; set; }
+        [ActivityInput(SupportedSyntaxes = new[] { SyntaxNames.JavaScript, SyntaxNames.Liquid })] 
+        public TimeSpan TimeToLive { get; set; }
+        [ActivityInput(SupportedSyntaxes = new[] { SyntaxNames.JavaScript, SyntaxNames.Liquid })] 
+        public string? ReplyToSessionId { get; set; }
+        [ActivityInput(SupportedSyntaxes = new[] { SyntaxNames.JavaScript, SyntaxNames.Liquid })] 
+        public DateTime ScheduledEnqueueTimeUtc { get; set; }
+        [ActivityInput(DefaultSyntax = SyntaxNames.Json, SupportedSyntaxes = new[] { SyntaxNames.JavaScript, SyntaxNames.Liquid,SyntaxNames.Json },UIHint = ActivityInputUIHints.MultiLine)] 
+        public IDictionary<string,Object> UserProperties { get; set; } = new Dictionary<string, object>();
```

Comment:
Should we move these to the Advanced category?

Reply:
yep 👍 

done!